test/org/sagebionetworks/bridge/play/controllers/ParticipantControllerTest.java

```diff
@@ -1109,10 +1110,8 @@ public void updateIdentifierRequiresAuthentication() throws Exception {
     
     @Test
     public void getParticipantByHealthCode() throws Exception {
-        AccountId accountId = AccountId.forHealthCode(study.getIdentifier(), "hc");
-        
         StudyParticipant studyParticipant = new StudyParticipant.Builder().withFirstName("Test").build();
-        when(mockParticipantService.getParticipant(study, accountId, true)).thenReturn(studyParticipant);
+        when(mockParticipantService.getParticipant(study, "healthCode:hc", true)).thenReturn(studyParticipant);
```

Comment:
There's probably no need to keep these "get by health code" and "get by external ID" tests, since the logic has been moved to the service.

Reply:
Removed